README.md

```diff
@@ -12,6 +12,15 @@ It is used:
 
 **Note:** we cannot directly change or improve various software that implements ActivityPub within the scope of this Taskforce, for that, please file issues with the respective project's issue trackers, potentially cross referencing issues here.
 
+## Taskforce Leadership
+
+This taskforce is co-chaired by:
```

Comment:
```suggestion
**Note:** within the scope of this task force, we cannot directly change nor improve the various software that implements ActivityPub. For that, please log issues with the respective project's issue trackers, potentially cross referencing issues here.

## Task Force Leadership

This task force is co-chaired by:
```

Reply:
The existing text is a run-on sentence. `Taskforce, for` should not have a comma; at least, it should be a semicolon. 

The reason to move the `within...force` clause is to clarify what it applies to. As it stands, it could read as modifying `software` (i.s., `software that implements ActivityPub within the scope of this Taskforce`) rather than modifying `we cannot`.